step.yml

```diff
@@ -0,0 +1,209 @@
+title: Save Cache
+summary: Saves build cache using a cache key. This Step needs to be used in combination with **Restore Cache**.
```

Comment:
Dont't forget to review and update this summary and the description. Not everything applies to the S3-specific logic